src/shared/components/palette/case-link/components/write-linked-cases.component.html

```diff
@@ -1,3 +1,5 @@
+<br />
+<br />
```

Comment:
seems not a good patter to add <br /> like this way, use css padding or margin is better than this br.